
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 1d23671..efc1a9a 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -126,9 +126,10 @@ target_include_directories("${ayatana_appindicator_gtkver}" PUBLIC ${PROJECT_DEP
 target_include_directories("${ayatana_appindicator_gtkver}" PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
 target_include_directories("${ayatana_appindicator_gtkver}" PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 target_link_libraries("${ayatana_appindicator_gtkver}" ${PROJECT_DEPS_LIBRARIES})
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/lib${ayatana_appindicator_gtkver}.so" DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}")
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/lib${ayatana_appindicator_gtkver}.so.1" DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}")
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/lib${ayatana_appindicator_gtkver}.so.1.0.0" DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}")
+install(TARGETS "${ayatana_appindicator_gtkver}"
+        LIBRARY DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}"
+        RUNTIME DESTINATION "${CMAKE_INSTALL_FULL_BINDIR}"
+        ARCHIVE DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}")
 
 # AyatanaAppIndicator{,3}-0.1.gir
